tests/sentry/issues/test_ingest.py

```diff
@@ -164,3 +178,34 @@ def test(self) -> None:
             "location": event.location,
             "last_received": event.datetime,
         }
+
+
+@region_silo_test
+class SaveIssueOccurrenceToEventstreamTest(OccurrenceTestMixin, TestCase):  # type: ignore
+    def test(self) -> None:
+        # TODO: We should make this a platform event once we have one
```

Comment:
What's a platform event? Like the `load_data` we use in other tests?

Reply:
It's an event that we received via the ingestion consumer that @rjo100 is working on. We might rename them to `generic` events based on the other pr, but we'll see. Basically just needs to be not an error or transaction event.